src/amo/components/SecondaryHero/styles.scss

```diff
@@ -0,0 +1,65 @@
+@import '~amo/css/styles';
+
+.SecondaryHero {
+  color: $black;
+  display: grid;
+  grid-gap: 32px;
+  grid-template-columns: repeat(4, 1fr);
+  min-height: 216px;
```

Comment:
What were you trying to address with this? If I remove this rule, the current dev home page looks the same.

Reply:
I think I did this when I was playing around with flexbox, but now with the grid it does not seem to be needed anymore.